### DISCUSSION TITLE
THANK YOU for "taming" Windows 11's task bar

### DISCUSSION THREAD

**Discussion by NormandieBerke** [valinet/ExplorerPatcher#2997] [General]
This is the only place I could find to thank "valenet" for his work to make Windows 11 BEARABLE  (with a RIGHT SIDED taskbar). Thanks to him him as well for so diligently KEEPING UP with (and fixing "Explorer Pathcher after the goofy stuff that happens with some Windows 11 (Microsoft) "updates."  We just sent to him what we could afford ( 20€ ).

**Comment by haydengriz** [comment]
you could have used discord
